.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,101 @@
+name: Bug Report
+description: Report a bug or issue
+title: "[Bug]: "
+labels: ["bug", "triage"]
+assignees: []
+body:
+  - type: markdown
+    attributes:
+      value: "#### Thank you for reporting this issue. Please provide all the required information to help us understand and resolve the issue."
+  
+  - type: input
+    id: bug-description
+    attributes:
+      label: "Description"
+      description: "Provide a summary of the issue."
+      placeholder: "Example: Adyen Checkout API returns an unexpected error when..."
+    validations:
+      required: true
+
+  - type: textarea
+    id: reproduce-steps
+    attributes:
+      label: "Steps to reproduce"
+      description: "List all necessary steps to reproduce the issue."
+      placeholder: |
+        1. Install the Adyen API'...'
+        2. Create '...'
+        3. Run '...'
+        4. Observer error '...'
+    validations:
+      required: true
+
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: "Expected behavior"
+      description: "Describe what you expected to happen."
+      placeholder: "Example: The Adyen Checkout API should return a 200 status with the correct response payload."
+    validations:
+      required: true
+
+  - type: textarea
+    id: actual-behavior
+    attributes:
+      label: "Actual behavior"
+      description: "Describe what actually happened."
+      placeholder: "Example: The API returns a 500 internal server error."
+
+  - type: textarea
+    id: code-snippet
+    attributes:
+      label: "Code snippet or screenshots (if applicable)"
+      description: "Provide relevant code snippets or screenshots to illustrate the issue."
+      placeholder: "```js\n// Your code here\n```"
+
+  - type: input
+    id: adyen-api-version
+    attributes:
+      label: "Adyen Node API Library version"
+      description: "Specify the version of the Adyen API library you're using."
+      placeholder: "Example: 20.1.2"
+    validations:
+      required: true
+
+  - type: dropdown
+    id: operating-system
```

Comment:
Let's remove the operation-system requirement: I've never seen it being used as part of the debug journey

Reply:
On the contrary, I think this is relevant as bugs sometimes depends the environment, including the OS. Users may experience issues that is unique with an operating system.